tools/plugins/com.liferay.ide.portlet.core/plugin.xml

```diff
@@ -177,4 +177,15 @@
             id="com.liferay.ide.portlet.core.jsf.jsfLiferayPortletDescriptorHelper">
       </liferayDescriptorHelper>
    </extension>
+   <extension
+         point="com.liferay.ide.project.core.portletFrameworks">
+      <portletFramework
+            class="com.liferay.ide.portlet.core.spring.SpringPortletFramework"
+            description="The Spring MVC portlet that run inside of Lifreay Portlet"
```

Comment:
Lets use this for the description:

Enables development of Spring MVC web applications using Spring's own Portlet MVC Framework.